validation-test/compiler_crashers_2_fixed/sr11599.swift

```diff
@@ -11,5 +11,5 @@ struct B {
 }
 
 struct C {
-  @A @B var foo: Int // expected-error{{extraneous argument label 'wrappedValue:' in call}}
+  @A @B var foo: Int // expected-error{{composed wrapper type 'B' does not match former 'wrappedValue' type 'Int'}}
```

Comment:
I don't understand this diagnostic... what are the two types facing a mismatch? Which `wrappedValue` is `former 'wrappedValue'` referring to here?

Reply:
For property wrapper composition `@A @B` the type of `A.wrappedValue` must be equal to `B`. The generalization is that a composed wrapper type must match the `wrappedValue` type of the previous property wrapper. I would love to workshop this error message, and an educational note would be great too!

Reply:
I also want to add a note for this error message to have a note pointing to the `wrappedValue` declaration, but I need to add a new locator to support that so I figured it'd be better to do that in a separate PR

Reply:
Ok, that makes sense. I agree we should have some mention of `A` here; since `wrappedValue` itself is the standard name, it doesn't provide much context. For example (straw-man suggestion)

> composed wrapper type 'B' does not match the type of 'A.wrappedValue' which is 'Int'
> note: in a property wrapper composition, the type of an outer property wrapper's wrappedValue must match the type of the inner property wrapper

Addressing this in a subsequent PR SGTM. 😄

Reply:
I like that suggestion, thank you!